File: SNS.Web/src/app/content-management/comments/contracts/comment-reaction-summary.dto.ts
import { ProfileSnapshotDto } from '../../../profiles/profiles/contracts/profile-snapshot.dto';
import { ReactionType } from '../../../shared/contracts/reaction-type';

export interface CommentReactionSummaryDto {
    user: ProfileSnapshotDto;
    reactionType: ReactionType;
    reactedAt: string;
}

File: backend/SNS.Application/ContentManagement/Comments/CommentReactions/Contracts/CommentReactionSummaryDto.cs
using SNS.Application.Profiles.Profiles.Contracts;
using SNS.Domain.ContentManagement.Shared.Enums;

namespace SNS.Application.ContentManagement.Comments.CommentReactions.Contracts;

public sealed record CommentReactionSummaryDto(
    ProfileSnapshotDto User,
    ReactionType ReactionType,
    DateTime ReactedAt
);
